text/0000-macros-1.1.md

```diff
@@ -492,3 +520,9 @@ pub struct Foo {
   we can layer on features backwards-compatibly to get to macros 2.0. Right now,
   though, the delta between what this RFC proposes and where we'd like to is
   very small, and can get get it down to actually zero?
+
+* Eventually macro crates will want to be loaded both at compile time and
+  runtime, and this means that Cargo will need to understand to compile these
+  crates twice, once as `rustc-macro` and once as an rlib. Does Cargo have
```

Comment:
Couldn't `rustc` do that? I guess there would be complications related to host/target separation. Not sure if @nrc had any solution in mind.